LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 MOXY
```

Comment:
Can you change this line to be more inline to how we usually do in other repos?

e.g.: https://github.com/moxystudio/js-proper-url-join/blob/master/LICENSE#L3

@ivolimasilva we might need to change https://github.com/moxystudio/next-runtime-env as well

@ivolimasilva